go/api/command/BUILD.bazel

```diff
@@ -12,7 +12,7 @@ proto_library(
 go_proto_library(
     name = "command_go_proto",
     importpath = "github.com/bazelbuild/remote-apis-sdks/go/api/command",
-    proto = ":command_proto",
+    proto = ":cmd_proto",
```

Comment:
Why this change?

Reply:
This is something that gazelle did automatically. I'll revert it from this change, but perhaps that's a naming convention that gazelle thinks is correct.